### DISCUSSION TITLE
Display a non-interactive map interactive=false

### DISCUSSION THREAD

**Discussion by shay-te** [rnmapbox/maps#1162] [Q&A] [answered]
Hello all, can you please help me. 

The `javascript` support `interactive=false`. how can I achieve this with `react-native` version?

https://docs.mapbox.com/mapbox-gl-js/example/interactive-false/

Thank you, 
Shay

**Comment by ferdicus** [comment, marked as answer]
Unfortunately there isn't a one stop solution for this atm (PRs welcome), so atm you would have to initialize a `MapView` and set these to `false`.

```
zoomEnabled | bool | none | false | Enable/Disable zoom on the map
scrollEnabled | bool | true | false | Enable/Disable scroll on the map
pitchEnabled | bool | true | false | Enable/Disable pitch on map
rotateEnabled | bool | true | false | Enable/Disable rotation on map
```

That should do the trick

**Reply by shay-te** [reply]
Thank you for the help!

**Reply by Florian-crg** [reply]
I've settled all theses to false 
        zoomEnabled: false,
        scrollEnabled: false,
        pitchEnabled: false,
        rotateEnabled: false,
        dragPan: false,
        dragRotate: false,
        doubleClickZoom: false
 I first cant drag but I just have to scroll on the map and all become true I guess, since I can drag to move after the scroll have been made. 
 The javascript support interactive=false would be very nice.

**Reply by a-tokyo** [reply]
> Unfortunately there isn't a one stop solution for this atm (PRs welcome), so atm you would have to initialize a `MapView` and set these to `false`.
> 
> ```
> zoomEnabled | bool | none | false | Enable/Disable zoom on the map
> scrollEnabled | bool | true | false | Enable/Disable scroll on the map
> pitchEnabled | bool | true | false | Enable/Disable pitch on map
> rotateEnabled | bool | true | false | Enable/Disable rotation on map
> ```
> 
> That should do the trick

As of today (using v6) one needs to disable  scrollZoom as well.

```js
scrollZoom: false,
```

So the final solution becomes:
```js
zoomEnabled: false,
scrollEnabled: false,
scrollZoom: false,
pitchEnabled: false,
rotateEnabled: false,
dragPan: false,
dragRotate: false,
doubleClickZoom: false
```

**Comment by shay-te** [comment]
Thank you for the comment it solves it  :+1:

**Reply by ferdicus** [reply]
feel free to ask questions like these within out [gitter channel](https://gitter.im/react-native-mapbox-gl/Lobby), it's full of helpful people :)

**Comment by shay-te** [comment]
will do it next time. thank you

**Comment by Gnative** [comment]
Just as a side note, if you want a non interactive map it might be worth looking at Mapbox static maps api. You can do quite a lot with that API and it's less of a hit on your usage limits

[Mapbox Static API](https://docs.mapbox.com/api/maps/static-images/)
[Mapbox Static Map playground](https://docs.mapbox.com/playground/static/)
